RELEASE/scripts/autoscend.ash

```diff
@@ -1795,6 +1795,7 @@ boolean doTasks()
 	if(LM_bhy())						return true;
 
 	tophatMaker();
+	auto_checkTrainSet();
 	deck_useScheme("");
```

Comment:
Deck cheats for ore if trainset isn't configured, so trainset needs to be configured before we deck cheat.